app/components/ui/atoms/CalloutTextBox.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Box } from 'grid-styled'
+import { th } from '@pubsweet/ui-toolkit'
+
+const Frame = styled(Box)`
+  border: ${th('borderWidth')} ${th('borderStyle')} ${th('colorPrimary')};
+  background-color: ${th('colorPrimary')}20;
+`
+
+const CalloutTextBox = ({ onClose, children, ...props }) => (
+  <Frame mb={2} p={3} {...props}>
```

Comment:
It's worth noting that `p={3}` means that there is padding top, left, bottom & right of `24px`. So, unlike the designs:
![image](https://user-images.githubusercontent.com/15656538/47159606-23383f80-d2e6-11e8-9aee-ca45b1dd139b.png)

We don't have `18px` top and bottom. In fact we have `24px`:
![image](https://user-images.githubusercontent.com/15656538/47210238-57653c00-d38a-11e8-8f86-35d0e2da7b9a.png)

@chugginselifesciences are you fine with this? I imagine that @tamlyn made this call based on the fact that we don't have a `18px` measurement in our grid units (see [`space` in our theme](https://github.com/elifesciences/elife-xpub/blob/develop/client/elife-theme/src/index.js#L49-L59))

Reply:
I'm also still not 100% convinced about putting `mb={2}` onto components themselves, rather than letting the parent handle the spacing between each component on the page :thinking: